tasks/main.yaml

```diff
@@ -1,13 +1,10 @@
 ---
+- name: Add apt-repo for Debian/Ubuntu
+  include: add-apt-repo.yaml
+  when: ansible_pkg_mgr == "apt"
```

Comment:
Please do not create additional task files. Please insert the single task in here.